macros/unpack/get_source_values.sql

```diff
@@ -24,8 +24,8 @@
               "cast(" ~ dbt_project_evaluator.is_not_empty_string(node.description) | trim ~ " as " ~ dbt.type_boolean() ~ ")",
               "cast(" ~ node.config.enabled ~ " as " ~ dbt.type_boolean() ~ ")",
               wrap_string_with_quotes(node.loaded_at_field | replace("'", "_")),
-              "cast(" ~ ((node.freshness != None) and (dbt_project_evaluator.is_not_empty_string(node.freshness.warn_after.count) 
-                or dbt_project_evaluator.is_not_empty_string(node.freshness.error_after.count))) | trim ~ " as boolean)",
+              "cast(" ~ ((node.config.freshness != None) and (dbt_project_evaluator.is_not_empty_string(node.config.freshness.warn_after.count) 
```

Comment:
actually, i'm realizing this logic should probably be:
```
node.config.freshness

OR 

node.freshness
```

before `node.freshness` is actually fully deprecated 

and same for below